src/AutocompleteBadges.js

```diff
@@ -0,0 +1,142 @@
+'use strict';
+
+import templates from './AutocompleteBadges.soy.js';
+import Autocomplete from 'metal-autocomplete';
+import Component from 'metal-component';
+import Soy from 'metal-soy';
+
+class AutocompleteBadges extends Component {
+
+	/**
+	 * @inheritDoc
+	 */
+	attached() {
+		this.autocomplete_ = new Autocomplete({
+			inputElement: this.refs.input,
+			data: this.getFilteredElements_.bind(this)
+		});		
+
+		this.autocomplete_.on('select', this.onListItemSelected_.bind(this));
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	rendered() {
+		if(this.refs.listBadges) {
```

Comment:
Maira,

I did this changes!
I removed the logic of set padding from js and now using CSS for this.
Thanks for the tips and advices! :D